### PR TITLE
Publication documentation

### PR DESCRIPTION
**Pull request by RhysJones125**
Adding details on how to create a bucket, connect to it within python, push files and get files

### CODE REVIEW

**Review by adrianclay — APPROVED** [review]
Nice.

File: dashboard_publication.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+bucket.upload_file(key='mykey.txt', filename='mykey.txt')
+```
+
+#### Accessing a file within the bucket
+
+As we are no longer wanting to store CSV's within the repo, we need to make a request to our s3 backing service to get out data.
+Pandas ```read_csv(...)``` is able to read directly from a file-like object such as our expected response from S3, which we get using ```s3.Object(...).get()['Body]```.
+
+```python
+response_content = s3.Object('paas-s3-broker-prod-lon-XXXX', "mykey.txt").get()['Body']
```
[the remark this answers is not in the document]
Comment (RhysJones125) [reply dashboard_publication.md:162]: Sounds like a good idea, thanks

File: dashboard_publication.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+cf service-key SERVICE_NAME SERVICE_KEY
+```
+
+---
+### Using a private bucket
+
+#### Connecting to s3
+In order to connect to s3, you will need an AWS access key and an AWS secret access key.
+
+#####To get credentials for the APP to the backing service use:
```
Comment (MattSutton17) [inline dashboard_publication.md:64]: Very minor, looks like this line doesn't render quite as expected - not sure it goes to 5th level heading?

**Review by MattSutton17 — APPROVED** [review]
Looks clear, nicely organised :)

File: dashboard_publication.md (hunk 2 above)
Comment (RhysJones125) [reply dashboard_publication.md]: Looks like it just needed a space between the hashes and the text. Thanks for pointing it out! PyCharm has failed me! It showed it all as fine :(

**Pull request merged**